src/rest-client.ts

```diff
@@ -2886,6 +2922,56 @@ export class RestClient extends BaseRestClient {
     );
   }
 
+  /**
+   *
+   * Financial product - Flexible loan endpoints
+   *
+   */
+
+  getBorrowableCurrencies(): Promise<
+    {
+      borrowCcy: string;
+    }[]
+  > {
+    return this.get('/api/v5/finance/flexible-loan/borrow-currencies');
+  }
+
+  getCollateralAssets(params?: {
+    ccy?: string;
+  }): Promise<CollateralAssetsResponse[]> {
```

Comment:
Are you sure about the return type is also an array? Looks like CollateralAssetsResponse is:
```typescript
export interface CollateralAssetsResponse
```

Might be cleaner to lift the top-level type, like this (double check the type is correct though please):
```
  getCollateralAssets(params?: {
    ccy?: string;
  }): Promise<{
  assets: CollateralAsset[];
  }> {
```